discovery-provider/alembic/versions/436c10e54758_create_profile_challenges.py

```diff
@@ -19,42 +18,34 @@
 
 
 def upgrade():
-    # ### commands auto generated by Alembic - please adjust! ###
     connection = op.get_bind()
-
-    # Create the challenges if needed
-    session = Session(bind=connection)
```

Comment:
@piazzatron if it's necessary to keep this in the migration, we'll have to pull in the functionality at the time of creation of this migration so that we don't have aggregate challenges attempting to be added before the schema migration occurs to allow that enum value. Is it ok to remove this or do we need to do that?

Reply:
chatted on slack - replacing this with a sql statement to create the profile-completion challenge, which is needed to create the user_challenges in this migration

Reply:
@piazzatron Added, please review at your leisure